rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -260,5 +270,41 @@ def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
             ELFI_LOCKING: (self._decode_elfi_claim,),
         }
 
-    def counterparties(self) -> list[str]:
-        return ETHEREUM_AIRDROPS_LIST
+    def counterparties(self) -> list[CounterpartyDetails]:
+        return [
+            CounterpartyDetails(
```

Comment:
For the uniswap case we have each version and the airdrop has its own too since it doesn't belong to any specific version. That is why it is not reused